.gitignore

```diff
@@ -16,3 +16,4 @@ data/pull-secret
 bin/
 dist
 .coverage
+data/*
```

Comment:
With this addition, we can probably remove `data/pull-secret` (line 15)... just cosmetic change though...

Reply:
done